### PR TITLE
Email verification adjustment

### PR DESCRIPTION
**Pull request by luwol03**
|               Status                |                Type                 | Env Vars Change |
| :---------------------------------: | :---------------------------------: | :-------------: |
| ✅ Ready | Feature |    No      |

## Description

This PR add's:
- CLI flags to disable/enable and verifyEmail of users.
- Disabled users cannot login
- expose verification configuration via info route
- expose created at of user

```json
{
    "identifier": "vocascan-server",
    "version": "1.2.1-rc.1",
    "locked": false,
    "commitRef": "9b0f913",
    "email_confirm": {
        "enabled": true,
        "level": "medium",
        "time": 1209600000
    }
}
```

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

File: app/Controllers/AuthController.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ const login = catchAsync(async (req, res) => {
 });
 
 const profile = catchAsync(async (req, res) => {
-  res.send(deleteKeysFromObject(['roleId', 'email', 'password', 'createdAt', 'updatedAt'], req.user.toJSON()));
```
Comment (noctera) [inline app/Controllers/AuthController.js:74]: In the Frontend you are using the "createdAt" property. You deleted the wrong one

File: app/Services/AuthServiceProvider.js [hunk 2]
```diff
@@ -131,7 +131,7 @@ async function createUser({ username, email, password, emailVerified, disabled }
   const isAdmin = await checkIfAdmin(user.id);
   const tempUser = { ...user.toJSON(), isAdmin };
 
-  return deleteKeysFromObject(['roleId', 'email', 'password', 'createdAt', 'updatedAt'], tempUser);
```
Comment (noctera) [inline app/Services/AuthServiceProvider.js:134]: Same

File: app/Services/AuthServiceProvider.js [hunk 3]
```diff
@@ -159,7 +159,7 @@ async function loginUser({ email, password }) {
...
```
Comment (noctera) [inline app/Services/AuthServiceProvider.js:162]: Same

File: app/Controllers/AuthController.js (hunk 1 above)
Comment (luwol03) [reply app/Controllers/AuthController.js:74]: No, these changes are correct. We now no longer delete the createdAt from the response.

Comment (noctera) [reply app/Controllers/AuthController.js:74]: Oh sorry, my bad. I have read over the "deleteKeysFromObject" function

**Review by noctera — APPROVED** [review]

**Pull request merged**